lib/vagrant-google/action/run_instance.rb

```diff
@@ -93,7 +94,8 @@ def call(env) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
           env[:ui].info(" -- Auto Restart:    #{auto_restart}")
           env[:ui].info(" -- On Maintenance:  #{on_host_maintenance}")
           env[:ui].info(" -- Autodelete Disk: #{autodelete_disk}")
-          env[:ui].info(" -- Scopes:          #{service_accounts}")
+          env[:ui].info(" -- Scopes:          #{service_accounts}") if service_accounts
```

Comment:
@seanmalloy Can you rename this to `scopes` or `service_account_scopes`?

This should help to avoid confusion in the long run.